com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -396,6 +399,10 @@ static float CalculateSurfaceArea(Matrix4x4 transform, Mesh mesh)
         static void DilateInvalidProbes(Vector3[] probePositions,
             List<Brick> bricks, SphericalHarmonicsL2[] sh, float[] validity, ProbeDilationSettings dilationSettings)
         {
+            if (!dilationSettings.dilate)
```

Comment:
While this would be the right thing to do I observed that _without_ dilation at all the result are hideous with current subdiv to the point that we shouldn't allow that. Did you try with the new subdiv? (the template_1 from the APV_Test repo is a good one)

Reply:
Ping on this. If it does look as hideous as it did before, we might need to _not_ do this change and let the dilation happen all the time (or at least make it default to ON)

Reply:
Yes, I removed it